### PR TITLE
Convert licenses to SPDX expressions

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**
To be merged once https://github.com/alire-project/alire/pull/629 is merged.

### CODE REVIEW

File: index/li/liblzma/liblzma-external.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 description = "The GNU Readline Library"
 name = "liblzma"
-licenses = ["GPL 2.0", "GPL 3.0", "LGPL 2.1"]
+licenses = "GPL-2.0-or-later AND GPL-3.0-or-later AND LGPL-2.1-or-later"
```
Comment (mosteo) [inline index/li/liblzma/liblzma-external.toml:3]: My understanding from reading SPDX docs [here](https://spdx.github.io/spdx-spec/appendix-IV-SPDX-license-expressions/) is that when you're giving the choice the appropriate operator is `OR`.

File: index/ze/zeromq_ada/zeromq_ada-4.1.5.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name = "zeromq_ada"
 version = "4.1.5"
 website = "https://github.com/persan/zeromq-Ada"
 authors = ["Per Sandberg <per.s.sandberg@bahnhof.se>"]
-licenses = ["MIT", "GMGPL 2.0"]
+licenses = "MIT AND GPL-2.0-only"
```
Comment (mosteo) [inline index/ze/zeromq_ada/zeromq_ada-4.1.5.toml:6]: Same thing here. Although it's possible both licenses apply and then `AND` is correct. Hi @reznikmm, are these licenses up for choice or do both apply simultaneously (due to reused code perhaps?)

**Comment by mosteo** [comment]
Sorry for the delay, I think the review got stuck in an unsent state...

File: index/li/liblzma/liblzma-external.toml (hunk 1 above)
Comment (Fabien-Chouteau) [reply index/li/liblzma/liblzma-external.toml:3]: liblzma licensing is a tad complicated: https://git.tukaani.org/?p=xz.git;a=blob;f=COPYING

Maybe the library itself in public domain so we can only use that one here.

Comment (mosteo) [reply index/li/liblzma/liblzma-external.toml:3]: All right, from reading that, indeed the proper operator would have been AND. But, as you say, the compiled bits seem to be all public domain.

File: index/ze/zeromq_ada/zeromq_ada-4.1.5.toml (hunk 2 above)
Comment (Fabien-Chouteau) [reply index/ze/zeromq_ada/zeromq_ada-4.1.5.toml:6]: I asked the author here: https://github.com/persan/zeromq-Ada/issues/17

Comment (mosteo) [reply index/ze/zeromq_ada/zeromq_ada-4.1.5.toml:6]: Thanks, Fabien.

**Comment by Fabien-Chouteau** [comment]
Looks like I cannot delete the `devel-05-spdx` branch, can you?

**Comment by mosteo** [comment]
> Looks like I cannot delete the `devel-05-spdx` branch, can you?

Right, `devel-` branches are protected. I temporarily removed the protection to delete it.

**Pull request merged**